HerPublicWebsite/Models/Questionnaire/PendingViewModel.cs

```diff
@@ -1,23 +1,27 @@
 using GovUkDesignSystem.Attributes.ValidationAttributes;
 using HerPublicWebsite.Models.Enums;
 using System.ComponentModel.DataAnnotations;
+using GovUkDesignSystem.Attributes;
+using GovUkDesignSystem.ModelBinders;
+using Microsoft.AspNetCore.Mvc;
 
 namespace HerPublicWebsite.Models.Questionnaire;
 
+
+
 public class PendingViewModel : QuestionFlowViewModel
 {
+
     public string LocalAuthorityName { get; set; }
     
     public string LocalAuthorityMessagePartialViewPath { get; set; }
 
-    [EmailAddress(ErrorMessage = "Enter an email address in the correct format, like name@example.com")]
-    [GovUkValidateRequiredIf(ErrorMessageIfMissing = "Enter your email address", IsRequiredPropertyName = nameof(IsEmailAddressRequired))]
-    public string EmailAddress { get; set; }
-
-    [GovUkValidateRequired(ErrorMessageIfMissing = "Select whether you would like to be contacted about future grants")]
-    public YesOrNo? CanContactByEmailAboutFutureSchemes { get; set; }
-
-    public bool IsEmailAddressRequired => CanContactByEmailAboutFutureSchemes is YesOrNo.Yes;
+    //This property is here as the Compare validation attribute requires an attribute to allow for only true values to be accepted.
+    public bool isTrue => true;
+    
+    [ModelBinder(typeof(GovUkCheckboxBoolBinder))]
+    [Compare("isTrue", ErrorMessage = "You must acknowledge that your application will not be processed until your local authority has signed up to use the service")]

```

Comment:
I think something like
```suggestion
    [Range(typeof(bool), "true", "true", ErrorMessage="...")]
```
might work? Which would save us the extra property. And if so then it might be nice to extract it to a custom attribute?

Otherwise let's use `nameof(isTrue)` to keep us robust to renames.

Reply:
I didn't realise you could use the Range validation attribute for booleans! It does seem to work when I've been testing it.
I've changed this over, but I can't figure out how to cleanly separate it into its own custom attribute.

Reply:
Happy to jump on a call to go through it if you want! Don't think this needs to be a blocker though

Reply:
Though I've just seen Jamie's spoilers, I would be interested in also knowing how to implement it as a custom attribute!

Reply:
Sure, we can chat about it tomorrow if you like